tests/Microsoft.Identity.Test.Unit/BrokerTests/RuntimeBrokerTests.cs

```diff
@@ -153,7 +188,115 @@ public void HandleInstallUrl_Throws()
         {
             AssertException.Throws<NotImplementedException>(() => _wamBroker.HandleInstallUrl("http://app"));
         }
+
+        // Done - Logs in one
+        // NA - logs in multiple
+        // Done - maps correctly
+        // Done - Is Pii enabled
+        // Done - does not emit when loglevel is not appropriate
+        // Is API Platform logging enabled?
+        // no mem leaks
+
+        [TestMethod]
+        public async Task ATS_CallsLog_When_CalledAsync()
```

Comment:
We have ROPC integration tests. Can you add real logging there and maybe assert that something comes from MSAL C++?

Reply:
Added